packages/core/src/service.ts

```diff
@@ -0,0 +1,180 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
```

Comment:
In LB4 documentation, we talk about services in many different ways, so i find this PR confusion somewhat.

`web services`: 

https://loopback.io/doc/en/lb4/Calling-other-APIs-and-web-services.html#add-a-service

`Remote and local services`:

https://loopback.io/doc/en/lb4/Service-generator.html#remote-and-local-services

`Connecting to SOAP services` https://loopback.io/doc/en/lb4/Connecting-to-SOAP.html 


`Add a Service` https://loopback.io/doc/en/lb4/soap-calculator-tutorial-add-service.html#add-a-service 
(talks about soap service here)

`Service` is not a key concept listed in our documentation either:

![image](https://user-images.githubusercontent.com/6864736/68485492-6759f300-020d-11ea-9695-b4d1c2e6b61e.png)

Reply:
I think this may be the same question as have been discussed in https://github.com/strongloop/loopback-next/pull/3489#issuecomment-523982645